liblouis/liblouis.h.in

```diff
@@ -478,6 +478,16 @@ LIBLOUIS_API
 void EXPORT_CALL
 lou_free(void);
 
+/**
+ * Free a pointer allocated by liblouis.
+ *
+ * This is a safe alternative for just calling free,
+ * which is especially helpful for freeing memory from foreign languages, like Python.
+ */
+LIBLOUIS_API
+void EXPORT_CALL
+lou_freePtr(void *);
```

Comment:
I think I'm more in favor of creating a dedicated "free" function for each API function that returns pointers to memory that must be freed by the caller. This generic free function seems so pointless (even though I understand the need for it).

Reply:
I think that makes sense to me.